index.bs

```diff
@@ -1131,14 +1133,14 @@ to filling the [=readable stream=]'s [=internal queue=] or changing its state. I
  The <dfn id="default-reader-closed" attribute for="ReadableStreamDefaultReader">closed</dfn>
  getter steps are:
 
- 1. Return [=this=].\[[closedPromise]].
+ 1. Return [=this=].[=ReadableStreamDefaultReader/[[closedPromise]]=].
```

Comment:
I refer to the specific class's `[[closedPromise]]` where possible, and only refer to the generic one when both types of readers could be passed in.

Not sure if this is worth the trouble? Maybe we should remove the `<dfn>\[[closedPromise]]</dfn>` on the specific readers, and only refer to the generic one? But then what should we put in the internal slots of the specific reader classes?

Reply:
Yeah, per my above comment, if we're going to collapse them, then I think we'd best collapse them fully.